engines/gob/console.cpp

```diff
@@ -67,8 +89,11 @@ bool GobConsole::cmd_dumpVars(int argc, const char **argv) {
 
 	Common::DumpFile file;
 
-	if (!file.open("variables.dmp"))
-		return true;
+    if (!file.open("variables.dmp")) {
+        const char *variablesdmp = "variables.dmp";
+        debugPrintf("Dumped %s successfully to ScummVM directory\n", variablesdmp);
+        return true;
+    }
```

Comment:
This has been edited incorrectly. What *should* be done is something like this:

```suggestion
    const char *variablesdmp = "variables.dmp";
    if (!file.open(variablesdmp ))
        return true;
    
    // ... dumping code remains the same ...
   
    debugPrintf("Dumped %s successfully to ScummVM directory\n", variablesdmp);
```

Reply:
fixed and commited